themes/cclw/constants/faqs.tsx

```diff
@@ -81,11 +81,8 @@ export const FAQS: TFAQ[] = [
           <li>View your search term (and related phrases) highlighted in search results</li>
           <li>Browse country profiles to find and compare their climate laws, policies and strategies</li>
           <li>
-            Access the raw data: you just need to{" "}
-            <ExternalLink url="https://docs.google.com/forms/d/e/1FAIpQLSdFkgTNfzms7PCpfIY3d2xGDP5bYXx8T2-2rAk_BOmHMXvCoA/viewform">
-              fill out this form
-            </ExternalLink>{" "}
-            to request a copy of the entire dataset
+            Access the raw data: you just need to <ExternalLink url="https://form.jotform.com/233131638610347">fill out this form</ExternalLink> to
```

Comment:
Why as the {" "} been removed?

Reply:
See below.

Reply:
![image](https://github.com/climatepolicyradar/navigator-frontend/assets/46493669/9444c8c2-78a3-45c0-a9b2-ed735b6d0047)